### PR TITLE
GSoC 2019 Proposal LIGO HTCondor Transfer limits proposal

### PR DESCRIPTION
**Pull request by efajardo**

### CODE REVIEW

File: _gsocproposals/2019/proposal_HTCondorTransferTests.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ * Replicate the tests of the HTCondor transfer method that were done for GlueX [1].
+ * Contribute to the [OSG Scalability Code repository|https://sourceforge.net/projects/osgscal/] and more specifically for the synthetic load generators:
+    1. Update the code to do  checksums of input and output transfers
+    1. Include Condor configuration for caputring and displaying transfer statistics
+    1. Capture transfer statistics (IO Transfer time, CPUTime) into InfluxDB using Condor built in Ganglia Daemon.
+ * Use and contribute to [glideTester|https://github.com/efajardo/osgscal/tree/master/glideTester] to evaluate the usage of a Cache, a set of caches with/without SSD up to 50000 parallel jobs.
+ * Combine the results of both and submit an abstract for CHEP Conference 2019.
+
+## Expected results
+Obtain conlcusive results on the current limitations of hardwared and software to support concurrency levels up to a five thousand jobs uploading or downloading data at different read and write speeds.
```
Comment (aflp91) [inline _gsocproposals/2019/proposal_HTCondorTransferTests.md]: Typo: 'conlcusive' => 'conclusive'

File: gsoc/2019/mentors.md [hunk 2]
```diff
@@ -68,5 +68,6 @@ layout: plain
 * Joao Calado Vicente [joao.calado.vicente@cern.ch](mailto:joao.calado.vicente@cern.ch) CERN
 * Tobias Wegner [tobias.wegner@cern.ch](mailto:tobias.wegner@cern.ch) CERN
 * Sandro Wenzel [sandro.wenzel@cern.ch](mailto:sandro.wenzel@cern.ch) CERN
+* Edgar Fajardo [emfajard@ucsd.edu](mailto:emfajard@ucsd.edu) UCSD
```
Comment (aflp91) [inline gsoc/2019/mentors.md]: Please, put mentors in alphabetical order.

**Comment by efajardo** [comment]
@aflp91 thanks for the review. I resolved the conflicts

File: _gsocproposals/2019/proposal_HTCondorTransferTests.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+year: 2019
+organization: UCSD
+---
+
+## Description
+
+The LIGO experiment is adopting the use of GlideinWMS, a distributed resource manager, for its computing needs. Currently, passing input data to the parallel jobs is accomplished via either HTCondor-based file transfer or downloading from a set of Xrootd-powered distributed caches. Past work has been done on testing the scalability of these transfer methods, but not in the context of the needs of the LIGO experiment. We will attempt to test at what concurrency levels different components of the GlideinWMS infrastructure break while keeping conditions aligned to the needs of the LIGO experiment. 
+
+
+## Task ideas
```
Comment (aflp91) [inline _gsocproposals/2019/proposal_HTCondorTransferTests.md]: Could you please fix the external links as `[Label](URL)`?

Comment (efajardo) [reply _gsocproposals/2019/proposal_HTCondorTransferTests.md]: @aflp91 sorry for that I update them.

**Pull request merged**